### ISSUE TITLE
Missing ext-pcntl for daemon mode

### ISSUE THREAD

**Issue by nupplaphil** [friendica/docker#72]
When I want to start the daemon through docker, the following error occurs:
```console
root@philipp /opt/friendica # docker-compose exec --user=www-data cron php bin/daemon.php start
Starting worker daemon.

Fatal error: Uncaught Error: Call to undefined function pcntl_fork() in /var/www/html/bin/daemon.php:124
Stack trace:
#0 {main}
  thrown in /var/www/html/bin/daemon.php on line 124
```

I guess the "native" forking possibility vanished from to another version...

**Comment by nupplaphil** [comment]
Fixed by #73

**Comment by annando** [comment]
The forking had always been the same. But you don't need to fork when you are running the daemon in the foreground.
